static/js/components/widgets/RelationField.test.tsx

```diff
@@ -209,6 +208,25 @@ describe("RelationField", () => {
       ).toEqual(formatWebsiteOptions(websites, "name"))
     })
 
+    //
+    ;[true, false].forEach(isCrossSite => {
+      it("should not filter on published if cross_site is not set", async () => {
```

Comment:
if you want, there's a neat facility built-in to Jest that you can use for parameterized tests like this, it looks something like this:

```suggestion
    it.each([true, false])("should not filter on published if cross_site is not set", async (isCrossSite) => {
```